lib/CSkillHandler.cpp

```diff
@@ -31,16 +31,9 @@ CSkill::LevelInfo::~LevelInfo()
 {
 }
 
-CSkill::CSkill(SecondarySkill id) : id(id)
+CSkill::CSkill(SecondarySkill id, std::string identifier) : id(id), identifier(identifier)
 {
-	if(id == SecondarySkill::DEFAULT)
-		identifier = "default";
-	else
-		identifier = NSecondarySkill::names[id];
-	// init levels
-	LevelInfo emptyLevel;
-	for(int level = 1; level < NSecondarySkill::levels.size(); level++)
-		levels.push_back(emptyLevel);
+	levels.resize(3);
```

Comment:
`3` should be global constant for now, more over higher skill level feature has been requested.